syntax = "proto3";

package wise.api;

service RocksDBService {
  rpc PutData (PutRequest) returns (PutResponse);
  rpc GetData (GetRequest) returns (GetResponse);
}

message PutRequest {
  string key = 1;
  string value = 2;
}

message PutResponse {
  bool success = 1;
}

message GetRequest {
  string key = 1;
}

message GetResponse {
  string value = 1;
  bool found = 2;
}
